ldap/servers/slapd/slapi2runtime.c

```diff
@@ -44,47 +46,49 @@
 Slapi_Mutex *
 slapi_new_mutex(void)
```

Comment:
Could consider inline attributes here.

Reply:
Not sure what you mean here, can you elaborate?